src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_buffer_fusing.cpp

```diff
@@ -191,7 +192,27 @@ bool concat_in_place_optimization::match(const program_node& concat_node,
             if (pred_l.format == format::b_fs_yx_fsv4 && (pred_l.feature() != 4 || concat_axis != 1))
                 return false;
         }
+
         if (pred.first->get_preferred_impl_type() == impl_types::onednn) {
+            // Onednn requires memory pointers to be aligned at least at 64-bytes to avoid potential correctness issues.
+            // https://uxlfoundation.github.io/oneDNN/dev_guide_c_and_cpp_apis.html#intel-r-processor-graphics-and-xe-architecture-graphics
```

Comment:
nit: Wouldn't it be better to refer to oneDNN documentation rather than including a URL that may change over time?

Reply:
I'm not sure what you mean by oneDNN documentation, but this link is the oneDNN documentation.

Reply:
Applied it.